apps/article/templates/amp_article.jade

```diff
@@ -0,0 +1,37 @@
+include amp_mixins
+include ../../../components/article/templates/mixins
+
+<!doctype html>
+html(amp)
+  head
+    +amp_head
+  body.body-article.body-responsive.body-amp
+    +amp_analytics
+    .amp-header: a.icon-logotype(href="#{sd.APP_URL}")
+    if article.get('hero_section')
+      +amp_hero_section(article.get('hero_section'))
```

Comment:
hmm just had an idea that we could move the more generic/boilerplate amp stuff to `main_layout` -- similar to how we sometimes use `blank`. would be easier for others to hop on.